ui/router.go

```diff
@@ -58,8 +58,8 @@ func initRouters() {
 	//API:
 	beego.Router("/api/search", &api.SearchAPI{})
 	beego.Router("/api/projects/:pid([0-9]+)/members/?:mid", &api.ProjectMemberAPI{})
-	beego.Router("/api/projects/", &api.ProjectAPI{}, "get:List")
-	beego.Router("/api/projects/?:id", &api.ProjectAPI{})
+	beego.Router("/api/projects/", &api.ProjectAPI{}, "get:List;post:Post")
```

Comment:
I don't think you need the post:Post chunk as this is the convention that maps "post" action to func "Post"